requirements.txt

```diff
@@ -13,6 +13,13 @@ django-settings-export>=1.2.1,<1.2.99
 django-macros>=0.4.0,<0.4.99
 django-debug-toolbar>=2.0,<2.0.99
 
+# Force this version of promise because the newest version doesn't support dict
+promise==2.2.1
```

Comment:
@justin0022, were we going to try to update this? We could delay, but we should make an issue.